src/test/java/com/google/javascript/clutz/DeclarationGeneratorTests.java

```diff
@@ -107,10 +108,16 @@ static String getExternFileNameOrNull(String testFileName) {
         getPackagePath().resolve("multifilePartial").toFile().listFiles(filter);
     // Output base files live in the 'outputBase' dir and impilicitly have base.js in their roots
     File[] testOutputBaseFiles = getPackagePath().resolve("outputBase").toFile().listFiles(filter);
+    // Test files that live in the 'crossModuleImports' dir, and run with the --partialInput option
+    // The resulting .d.ts files are checked with a DeclarationSyntaxTest, and they're also
+    // compiled in a single run in MultiFileTest
```

Comment:
Is this the reason to not put them alongside the other 'partial' tests -- that they're checked/compiled after?

Reply:
Right, they need to be in their own directory to do the total compile as well as the syntax check.